packages/react-composites/tests/app/lib/HiddenChatComposites.tsx

```diff
@@ -12,23 +12,67 @@ import { ChatAdapter, ChatComposite } from '../../../src';
  * The added chat composites are hidden, but tests can interact with them programmatically.
  */
 export const HiddenChatComposites = (props: { adapters: ChatAdapter[] }): JSX.Element => {
+  const [hiddenCompositesProps, setHiddenCompositesProps] = useState<HiddenChatCompositesProps[]>([]);
```

Comment:
maybe dumb question but what is the reason for the hidden composites again? just to send messages in the tests? (wont block this PR on this but would love to know why we have these)

Reply:
Yeah just to send messages and read receipts. They don't really have to be hidden tbh, and I wonder if we could replace them with just adapter uses and fire off read events and messages and such through the adapter interface

Reply:
yeah could be a interesting refactoring of the tests. I don't think its anything worth blocking this over though since are already using this.